Resources/Prototypes/Loadouts/satchels.yml

```
@@ -259,3 +259,17 @@
+  id: LoadoutBackpackCE
+  category: Backpacks
+  cost: 0
+  exclusive: true
+  items:
+    - ClothingBackpackCE
+  requirements:
+    - !type:CharacterItemGroupRequirement
+      group: LoadoutBackpacks
+    - !type:CharacterJobRequirement
+      jobs:
+        - ChiefEngineer
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)